keyboards/sck/neiso/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "NEISO",
+    "url": "",
+    "maintainer": "jrfhoutx",
+    "width": 9.25,
+    "height": 14.25,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"x":2.25, "y":0, "w":1.25, "h":2}, {"x":1.25, "y":1.25}, {"x":-4.75, "y":6.5, "w":1.25, "h":2}, {"x":8, "y":13.25, "w":1.25, "h":2}, {"x":2.75, "y":10.5, "w":1.25, "h":2}]
```

Comment:
Your implementation here is wonky because of the key rotation. (Our converter gives weird output under that condition.)

![image](https://user-images.githubusercontent.com/18669334/67628949-9fcff900-f82b-11e9-810a-e5f80238671a.png)

Try this:

```suggestion
            "layout": [{"x":0.25, "y":0, "w":2, "h":1.25}, {"x":2.75, "y":0, "w":1.25, "h":2}, {"x":1.5, "y":1.5}, {"x":0, "y":1.75, "w":1.25, "h":2}, {"x":1.75, "y":2.75, "w":2, "h":1.25}]
```